Library/Homebrew/bintray.rb

```diff
@@ -56,16 +56,27 @@ def official_org?(org: @bintray_org)
   end
 
   def create_package(repo:, package:, **extra_data_args)
-    url = "#{API_URL}/packages/#{@bintray_org}/#{repo}/#{package}"
+    url = "#{API_URL}/packages/#{@bintray_org}/#{repo}"
     data = { name: package, public_download_numbers: true }
     data[:public_stats] = official_org?
     data.merge! extra_data_args
-    open_api url, "--request", "POST", "--data", data.to_json
+    open_api url, "--header", "Content-Type: application/json", "--request", "POST", "--data", data.to_json
   end
 
   def package_exists?(repo:, package:)
     url = "#{API_URL}/packages/#{@bintray_org}/#{repo}/#{package}"
-    open_api url, "--output", "/dev/null", auth: false
+    begin
+      open_api url, "--silent", "--output", "/dev/null", auth: false
+    rescue ErrorDuringExecution => e
+      stderr = e.output.select { |type,| type == :stderr }
+                .map { |_, line| line }
```

Comment:
```suggestion
      stderr = e.output
                .select { |type,| type == :stderr }
                .map { |_, line| line }
```

or, if it works:
```suggestion
      stderr = e.output
                .select { |type,| type == :stderr }
                .values
```

Reply:
The second option won't work because  `.select { |type,| type == :stderr }` returns an array.
Will create a PR with the first one